src/structures/idt.rs

```diff
@@ -735,8 +735,9 @@ impl<F> Entry<F> {
         &mut self.options
     }
 
+    /// Returns the virtual address of this IDT entry's handler function as a `u64`.
     #[inline]
-    fn handler_addr(&self) -> u64 {
+    pub fn handler_addr(&self) -> u64 {
```

Comment:
This should return a `VirtAddr`, if we're going to do this.

Reply:
sure, np. As mentioned in the PR summary, I'm fine with doing so if y'all think it's appropriate. :+1:

Reply:
done.